### ISSUE TITLE
NTR: Liver biopsy: beta-ureidopropionase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9257]
New term request Liver biopsy: beta-ureidopropionase activity low (elthat_111121053103): beta-alanine synthase low on hepatic biopsy  UMLS: 1847706

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000082
- label: Reduced hepatic beta-ureidopropionase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:15385443
- def: Activity of beta-ureidopropionase in the liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9257
- synonyms: Liver biopsy: beta-ureidopropionase activity low(NONE)
